### PR TITLE
Add missing custom-* data (part 3)

### PR DESCRIPTION
**Pull request by ApCoder123**
Part of #6845

### CODE REVIEW

File: entries/c/caltech.edu.json [hunk 1]
```diff
@@ -6,7 +6,10 @@
       "custom-software",
       "custom-hardware"
     ],
-    "documentation": "https://imss.caltech.edu/node/1408",
+    "custom-software": [
+      "Duo"
+    ],
```
Comment (Carlgo11) [inline entries/c/caltech.edu.json:11]: custom-hardware with Yubikey OTP can be added as well.

File: entries/c/cornell.edu.json [hunk 2]
```diff
@@ -8,6 +8,9 @@
       "u2f",
       "call"
     ],
+    "custom-software": [
+      "Duo"
+    ],
```
Comment (Carlgo11) [inline entries/c/cornell.edu.json:13]: Maybe remove the custom-hardware as u2f is used?

File: entries/d/duke.edu.json [hunk 3]
```diff
@@ -8,6 +8,9 @@
       "custom-software",
       "custom-hardware"
     ],
+    "custom-software": [
+      "Duo"
+    ],
```
Comment (Carlgo11) [inline entries/d/duke.edu.json:13]: Yubico OTP support as well

File: entries/w/washington.edu.json [hunk 4]
```diff
@@ -5,10 +5,15 @@
...
       "call",
       "custom-software",
-      "u2f"
+      "custom-hardware"
+    ],
+    "custom-software": [
+      "Duo"
+    ],
+    "custom-hardware": [
+      "Feitian OTP c100 Token"
```
Comment (Carlgo11) [inline entries/w/washington.edu.json]: >Any token that produces generic 6- or 8-digit OATH-HOTP passcodes or Yubico’s proprietary AES OTP passcodes is compatible

Yubico OTP can also be listed

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/c/cornell.edu.json (hunk 2 above)
Comment (ApCoder123) [reply entries/c/cornell.edu.json:13]: Looks like D100 tokens are supported as well https://it.cornell.edu/twostep/choose-your-authentication-device-two-step-login

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**